core/datainput.gms

```diff
@@ -1089,34 +1101,31 @@ $offdelim
 p_histCapFac(tall,all_regi,"windon") $ (p_histCapFac(tall,all_regi,"windon") eq 0) = p_histCapFac(tall,all_regi,"wind");
 p_histCapFac(tall,all_regi,"wind") = 0;
 
-*** RP rescale wind capacity factors in REMIND to account for very different real-world CF 
-*** (potentially partially due to assumed low-wind turbine set-ups in the NREL data)
-*** Because of the lag effect (turbines in the 2000s were much smaller and thus yielded lower CFs),
-*** only implement half of the calculated ratio of historic to REMIND capFac as rescaling for the new CFs - realised as (x+1)/2
-
-*cb* CF calibration analogously for wind and spv: calibrate 2015, and assume gradual phase-in of grade-based CF (until 2045 for wind, until 2030 for spv)
-p_aux_capacityFactorHistOverREMIND(regi,"windon")  $ p_avCapFac2015(regi,"windon")  = p_histCapFac("2015",regi,"windon")  / p_avCapFac2015(regi,"windon");
-p_aux_capacityFactorHistOverREMIND(regi,"windoff") $ p_avCapFac2015(regi,"windoff") = p_histCapFac("2015",regi,"windoff") / p_avCapFac2015(regi,"windoff");
-
-*** linear phase-in of Remind capacity factors instead of historical ones
-loop(t$(t.val ge 2015 AND t.val le 2035),
-  pm_cf(t,regi,teWind) =
-    (2035 - pm_ttot_val(t)) / (2035-2015) * pm_cf(t,regi,teWind) * p_aux_capacityFactorHistOverREMIND(regi,teWind)
-    +
-    (pm_ttot_val(t) - 2015) / (2035-2015) * pm_cf(t,regi,teWind)
+
+*** Capacity factor for wind and solar
+*** Effective capacity factor pm_dataren("nur") * pm_cf scales from historical values in 2015 to grade-based values in 2030
+***   pm_dataren("nur",rlf) is the capacity factor of a given rlf grade
+***   pm_cf is a multiplier that scales linearly from p_aux_capacityFactorHistOverREMIND in 2015 to 1 in 2030
+*** This scaling accounts for lag effects, for instance turbines in the 2000s were much smaller hence yielding lower capacity factors
+p_aux_capacityFactorHistOverREMIND(regi,teVRE) = 1;
+p_aux_capacityFactorHistOverREMIND(regi,teVRE) $ (p_histCapFac("2015",regi,teVRE) and p_avCapFac2015(regi,teVRE)) =
+  p_histCapFac("2015",regi,teVRE) / p_avCapFac2015(regi,teVRE);
+
+loop(t $ (t.val ge 2015 AND t.val lt 2030),
```

Comment:
For consistency, all teVRE now scale their capacity factor between 2015 and 2030. Happy to change that if something else makes more sense @robertpietzcker. Also we could think of using 2020 historical cf instead of 2015.

Previously, it was different for each technology but i could not find why:
- spv scaled from 2015 to 2030.
- csp did not use historical cf.
- wind scaled from 2015 to 2035 (but comments said 2045).
- windoff scaled from 2015 to 2035 but with missing data causing the bug.